packages/react-widgets/src/WidgetPicker.tsx

```diff
@@ -4,16 +4,17 @@ import React from 'react'
 function WidgetPicker({
   className,
   hideCaret = false,
+  busy = false,
   ...props
-}: React.HTMLProps<HTMLDivElement> & { hideCaret?: boolean }) {
+}: React.HTMLProps<HTMLDivElement> & { hideCaret?: boolean, busy?: boolean }) {
   return (
     <div
       {...props}
       className={cn(
         className,
         'rw-widget-picker',
         'rw-widget-container',
-        hideCaret && 'rw-hide-caret',
+        (hideCaret && !busy) && 'rw-hide-caret',
```

Comment:
last thing!, i wouldn't conflate the meaning here as this isn't what you'd expect in other components. We can do this logic in Combobox where we are synthesizing the hide meaning more specifically